src/backend/move_logic.js

```diff
@@ -217,6 +217,18 @@ export function RegisterMoveCallback(roomName, turn, playerID, realPlayerName, s
 								setConfirmed(false);
 								setCurrentMove("blocked");
 								blocked = true;
+								if (doc.data().blocks[0] !== undefined){
+									if (doc.data().blocks[0].card === "Ambassador"){
+										setWaitingMessage(targetPlayer + " blocked your move as Ambassador. Pick one!")
```

Comment:
Can you change it to pick one card to keep?

Reply:
This is the pick one option: Let it go/ call bluff

Reply:
not ambassador move